spec/game_spec.rb

```diff
@@ -5,26 +5,30 @@
     describe '#score' do
```

Comment:
So i think the #score tests are great tests that are more overall.
Let's make unit tests for the individual methods.